src/android.ts

```diff
@@ -608,25 +637,25 @@ export const androidApi = <RunTarget extends SupportedRunTarget<'android'>>(
 
         if (await this._internal.hasCertificateAuthority(certFilename)) return;
 
-        await this._internal.requireRoot('installCertificateAuthority');
+        const { adbRootPush } = await this._internal.requireRoot('installCertificateAuthority');
 
         // Since Android 10, we cannot write to `/system` anymore, even if we are root, see:
         // https://github.com/tweaselORG/meta/issues/18#issuecomment-1437057934
         // Thanks to HTTP Toolkit for the idea to use a tmpfs as a workaround:
         // https://github.com/httptoolkit/httptoolkit-server/blob/9658bef164fb5cfce13b2c4b1bedacc158767f57/src/interceptors/android/adb-commands.ts#L228-L230
         await this._internal.overlayTmpfs('/system/etc/security/cacerts');
 
-        await adb(['push', path, `/system/etc/security/cacerts/${certFilename}`]);
+        await adbRootPush(path, `/system/etc/security/cacerts/${certFilename}`);
     },
     async removeCertificateAuthority(path) {
         const certFilename = `${await this._internal.getCertificateSubjectHashOld(path)}.0`;
 
         if (!(await this._internal.hasCertificateAuthority(certFilename))) return;
 
-        await this._internal.requireRoot('removeCertificateAuthority');
+        const { adbRootShell } = await this._internal.requireRoot('removeCertificateAuthority');
 
         await this._internal.overlayTmpfs('/system/etc/security/cacerts');
-        await adb(['shell', 'rm', `/system/etc/security/cacerts/${certFilename}`]);
+        await adbRootShell(['rm', `/system/etc/security/cacerts/${certFilename}`]);
     },
     async setProxy(_proxy) {
```

Comment:
Running the example script, the VPN never gets activated (I don't see the key icon in the top bar). But the check for the VPN doesn't fail, either.

Reply:
I don't see our config in the app.